packages/status/src/class-status.php

```diff
@@ -16,30 +16,61 @@ class Status {
 	/**
 	 * Is Jetpack in development (offline) mode?
 	 *
-	 * @return bool Whether Jetpack's development mode is active.
+	 * @deprecated 8.8.0 Use Status->is_offline_mode().
+	 *
+	 * @return bool Whether Jetpack's offline mode is active.
 	 */
 	public function is_development_mode() {
-		$development_mode = false;
-		$site_url         = site_url();
+		return $this->is_offline_mode();
```

Comment:
Should we add a deprecation notice here?

Reply:
My only hesitation was the comment on https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L1642

It'll probably be fine to deprecate it since it is a newer function. What do you think?

Reply:
Added it via b42b9cbb4

Reply:
Ah, good catch!

> It'll probably be fine to deprecate it since it is a newer function. What do you think?

Yes, that seems okay!